themes/bootstrap3/js/lib/status_ajax_queue.js

```diff
@@ -0,0 +1,63 @@
+class StatusAjaxQueue {
```

Comment:
Since this doesn't have anything specific to statuses and is used for versions as well, could we call it just `AjaxQueue` or `AjaxRequestQueue`?